packages/core/src/components/account/MyAccountDrawer/OrganizationDrawer/OrganizationDrawer.tsx

```diff
@@ -26,9 +26,9 @@ export const OrganizationDrawer = ({
   isRepresentative,
 }: OrganizationDrawerProps) => {
   const { fade, fadeOut } = useFadeEffect()
-  const { b2b } = useSession()
+  const { b2b, person } = useSession()
 
-  const contractName = b2b?.firstName ?? ''
+  const contractName = person?.givenName ?? ''
```

Comment:
I'm not sure about it. Is the `person.givenName` the contract name?
 
<img width="318" height="571" alt="Screenshot 2025-07-22 at 18 10 05" src="https://github.com/user-attachments/assets/7fc70a5f-c893-43fe-a905-f62af2727073" />

Maybe @manuellacorreia can help here.

Reply:
Yeah, the `person` info comes from the `profile` session namespace that the B2B team informed is related to contract 🤔